src/compiler/parser.ts

```diff
@@ -95,6 +95,980 @@ namespace ts {
         return isMetaProperty(node) && node.keywordToken === SyntaxKind.ImportKeyword && node.name.escapedText === "meta";
     }
 
+    type ForEachChildFunction = <T>(node: any, cbNode: (node: Node) => T | undefined, cbNodes?: (nodes: NodeArray<Node>) => T | undefined) => T | undefined;
+
+    const forEachChildTable: Array<ForEachChildFunction | undefined> = [];
+    for (let i = 0; i < SyntaxKind.Count; i++) forEachChildTable.push(undefined);

```

Comment:
As long as we're micro-optimizing, I wonder if it would be happier with a no-op function.

Reply:
There might've been something I missed, but it didn't seem to be.

Reply:
Interesting.  Maybe `undefined` is already in the domain.  Thanks for checking!